### ISSUE TITLE
When I get an `<html>` element with `first(:xpath, '..')`, the `matches_css?` method does not work.

### ISSUE THREAD

**Issue by gomo** [teamcapybara/capybara#2097]
## Meta
Capybara Version: 3.7.1

Driver Information (and browser if relevant): 
selenium-webdriver 3.11.0
Chrome 69.0.3497.81

## Expected Behavior

```rb
first('html').matches_css?('html') # true
first('body').first(:xpath, '..').matches_css?('html') # true
```

## Actual Behavior

```rb
first('html').matches_css?('html') # true
first('body').first(:xpath, '..').matches_css?('html') # false
```

## Steps to reproduce

```html
# public/simple.html
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta http-equiv="X-UA-Compatible" content="ie=edge">
  <title>Document</title>
</head>
<body>
  simple
</body>
</html>
```

```rb
visit('/simple.html')
binding.pry
```

```
[1] pry(#<RSpec::ExampleGroups::Selenium::Nested>)> first('html').matches_css?('html')
=> true
[2] pry(#<RSpec::ExampleGroups::Selenium::Nested>)> first('body').first(:xpath, '..').matches_css?('html')
=> false
```

**Comment by twalpole** [comment]
Hmmm, this is an edge case because the `html` element doesn't have a parent to use as the scope when looking for all `html` elements to see if any are the same element as the original `html` element.  We should probably be defaulting to the document if no parent is found rather than the previous query scope.  I'll look into it further in the morning.

**Comment by twalpole** [comment]
Fixed via 6971ac6967e4ac280583b5ec9d8ad8d9fe414cbd - Thanks for the clear issue report.

**Comment by gomo** [comment]
@twalpole 

> Fixed via 6971ac6

It's working as expected.
Thanks for the quick fix!
